sdk/eventhub/azure-eventhub-checkpointstoretable/MANIFEST.in

```
@@ -0,0 +1,5 @@
+include _meta.json
+include *.md
+include azure/__init__.py
```

missing a couple lines:
```suggestion
include *.md
include azure/__init__.py
include azure/eventhub/__init__.py
include azure/eventhub/extensions/__init__.py
```